<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentRegistrationScreen</class>
 <widget class="QMainWindow" name="StudentRegistrationScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1603</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleText">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>861</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Student Registration Portal</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ToggleFullScreen">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>30</y>
      <width>25</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>][</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEditField"/>
     </item>
     <item>
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailEditField"/>
     </item>
     <item>
      <widget class="QLabel" name="studentIdLabel">
       <property name="text">
        <string>Student ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="studentIdEditField"/>
     </item>
     <item>
      <widget class="QLabel" name="numberLabel">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numberEditField"/>
     </item>
     <item>
      <widget class="QLabel" name="departmentLabel">
       <property name="text">
        <string>Department:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="departmentEditField"/>
     </item>
     <item>
      <widget class="QPushButton" name="studentRegistrationButton">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
